azure-proto-compute/XAvailabilitySet.cs

```diff
@@ -4,7 +4,7 @@ namespace azure_proto_compute
 {
     public class XAvailabilitySet : AvailabilitySetOperations
     {
-        public XAvailabilitySet(ArmClientContext context, PhAvailabilitySet resource):base(context, resource)
+        public XAvailabilitySet(ArmClientContext context, PhAvailabilitySet resource, ArmClientOptions clientOptions): base(context, resource, clientOptions)
```

Comment:
not sure we want this here as this introduces the opportunity for people to "change" the opitions inside of a client instance